### DISCUSSION TITLE
All HttpClient request failed in Browser project

### DISCUSSION THREAD

**Discussion by ismlsmile** [AvaloniaUI/Avalonia#14946] [Q&A]
### Describe the bug

 I guess it is a Cross-Origin Resource Sharing (CORS) problem, but I think this should be transparent to the user. As I tried Flutter, it works well in web platform, and user do not need to do any special config, just as same as the desktop platform.

### To Reproduce

Try any http request

### Avalonia version

All

**Comment by maxkatz6** [comment]
Flutter on web will have the same CORS problems. Flutter with CanvasKit is more or less identical in the way how Avalonia works. 

CORS is a well-known browser limitation for web developers created to ensure security. There is plenty of information on that in the web. 

This one is my favorite: https://developer.mozilla.org/en-US/docs/Web/HTTP/CORS
You can find more information about configuring server side to enable your client-side host.
